delivery/repositories/runfolder_repository.py

```diff
@@ -107,3 +115,86 @@ def get_project(self, project_name):
         for project in self.get_projects():
             if project.name == project_name:
                 yield project
+
+    def dump_project_checksums(self, project):
+        raise NotImplementedError()
+
+    def samplesheet_file(self, runfolder):
+        return os.path.join(runfolder.path, self.SAMPLESHEET_PATH)
+
+    def checksum_file(self, runfolder):
+        return os.path.join(runfolder.path, self.CHECKSUM_FILE_PATH)
+
+    def get_samplesheet(self, runfolder):
+        return self.file_system_service.extract_samplesheet_data(self.samplesheet_file(runfolder))
```

Comment:
Not a bigge, but I wonder if perhaps it would make sense to place this code in some other module than the `file_system_service`. Considering that it is a static method it should be simple enough to drop it into it's own module.

Reply:
:+1: will do